app/helpers/dropdown/event/participant_add.rb

```diff
@@ -27,8 +32,9 @@ def user_participates_in?(user, event)
         end
       end
 
-      def initialize(template, group, event, label, icon, url_options = {})
+      def initialize(template, user, group, event, label, icon, url_options = {}) # rubocop:disable Metrics/ParameterLists
```

Comment:
Hast du mal kurz über alle Wagons gesucht, ob diese Klasse noch irgendwo instanziiert wird?

Reply:
Habe nachgeschaut, in unseren Wagons wird das sonst nirgends erwähnt.